README.markdown

```diff
@@ -279,23 +275,23 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons should have one space on the left and one on the right. Exceptions are empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```

Comment:
Maybe the one space on the left should be added as an exception since are more the cases with no space like specifying the variable/property type, conforming to protocols or subclassing.
The preferred examples have these incongruences to what was said:
`class TestDatabase: Database {` and `var data: [String: Float] = ... `

What do you think?